src/components/Card/Card.tsx

```diff
@@ -0,0 +1,48 @@
+import styles from './Card.module.css';
+
+export default function Card({
+  numbers,
+  cardLogo,
+  expiration
+}: {
+  numbers: string[];
+  cardLogo: 'visa' | 'master' | '';
```

Comment:
```suggestion
  cardLogo: CardLogo;
```
아까 위에서 선언한 타입을 여기서 재사용할 수 있을 것 같아요🙃